### PR TITLE
ADEN-8230 Register skin template in AE3

### PR DESCRIPTION
**Pull request by fraszczakszymon**

### CODE REVIEW

File: extensions/wikia/AdEngine3/src/templates/background-changer.js [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+			document.querySelector('.background-image-gradient').classList.add('no-gradients');
+		}
+	} else {
+		document.body.classList.remove('background-dynamic');
+		document.body.classList.remove('background-not-tiled');
+		document.body.classList.remove('background-fixed');
+	}
+}
+
+class BackgroundChanger {
```
Comment (nandy-andy) [inline extensions/wikia/AdEngine3/src/templates/background-changer.js:19]: Inspired by https://github.com/Wikia/app/blob/61d6d1b394768cf9ea78a22488d40750ce7eec35/skins/oasis/js/BackgroundChanger.js :)

File: extensions/wikia/AdEngine3/src/templates/skin.js [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+			pixelElement = document.createElement('img');
+			pixelElement.src = pixelUrl;
+			pixelElement.width = 1;
+			pixelElement.height = 1;
+			parentNode.appendChild(pixelElement);
+		}
+	});
+}
+
+export class Skin {
```
Comment (nandy-andy) [inline extensions/wikia/AdEngine3/src/templates/skin.js:20]: Inspired by https://github.com/Wikia/app/blob/b34866eef012d99285c732311d60fcf7b541df03/extensions/wikia/AdEngine/js/template/skin.js ;)

**Review by nandy-andy — APPROVED** [review]

File: extensions/wikia/AdEngine3/src/templates/background-changer.js (hunk 1 above)
Comment (fraszczakszymon) [reply extensions/wikia/AdEngine3/src/templates/background-changer.js:19]: Yes, I didn't want to hack and use original backgroundChanger attached to window. I believe we can remove BackgroundChanges.js once we finish with AE3. It seems that it's used only by AE2.

File: extensions/wikia/AdEngine3/src/templates/skin.js (hunk 2 above)
Comment (fraszczakszymon) [reply extensions/wikia/AdEngine3/src/templates/skin.js:20]: :+1:

**Pull request merged**